features/layer_lock.h

```diff
@@ -82,7 +82,21 @@ void layer_lock_invert(uint8_t layer);
 //   }
 void layer_lock_set_user(layer_state_t locked_layers);
 
+#if LAYER_LOCK_IDLE_TIMEOUT > 0
+
+// If you want to automatically disable layer lock if no keys are pressed after a
+// configurable timeout, define LAYER_LOCK_IDLE_TIMEOUT to a value between 100
+// and 30000 (in milliseconds) in your config.h file and add the following
```

Comment:
Now that process_layer_lock_time() has been absorbed, update this comment to something like: "...and call layer layer_lock_timer_task() from your matrix_scan_user function()`.

Reply:
Those directions are directly above the layer_lock_timer_task. Do you think I should move them above and put it all in the same block of instructions?

```
    void layer_lock_all_off(void);

// To enable the layer_lock timer you will have to add the layer_lock_timer_task
// function under your matrix_scan_user function, most likely in keymap.c.
    void layer_lock_timer_task(void);
```